Sources/SpeziOpenAI/MessagesView.swift

```diff
@@ -67,32 +66,31 @@ public struct MessagesView: View {
     /// - Parameters:
     ///   - chat: The chat messages that should be displayed.
     ///   - bottomPadding: A fixed bottom padding for the messages view.
-    ///   - hideSystemMessages: If system messages should be hidden from the chat overview.
+    ///   - hideMessagesWithRoles: If .system and/or .function messages should be hidden from view.
     public init(
         _ chat: [Chat],
-        bottomPadding: CGFloat = 0,
-        hideSystemMessages: Bool = true
+        hideMessagesWithRoles: Set<Chat.Role>,
+        bottomPadding: CGFloat = 0
     ) {
         self._chat = .constant(chat)
+        self.hideMessagesWithRoles = hideMessagesWithRoles
         self._bottomPadding = .constant(bottomPadding)
-        self.hideSystemMessages = hideSystemMessages
     }
-    
-    
+
     /// - Parameters:
     ///   - chat: The chat messages that should be displayed.
-    ///   - bottomPadding: A bottom padding for the messages view.
-    ///   - hideSystemMessages: If system messages should be hidden from the chat overview.
+    ///   - bottomPadding: A fixed bottom padding for the messages view.
```

Comment:
If you can pass the padding is a binding, it is not fixed but variable:
```suggestion
    ///   - bottomPadding: A bottom padding for the messages view.
```